src/components/AddActivityPage.js

```diff
@@ -1,26 +1,35 @@
 import React from 'react';
-import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 import AcivityForm from './ActivityForm';
 import { addActivity } from '../actions/activities';
 import { ACTIVITY_PLAN_ROUTE } from '../constants/routes';
 
-const AddActivityPage = ({onAddActivity, history}) => (
+const AddActivityPage = ({ onAddActivity, history }) => (
   <div>
     <h1>Add Activity</h1>
     <AcivityForm
-      onSubmit={(activity) => {
+      onSubmit={activity => {
         onAddActivity(activity);
         history.push(ACTIVITY_PLAN_ROUTE);
       }}
     />
   </div>
 );
 
-const mapDispatchToProps = ({
+AddActivityPage.propTypes = {
+  onAddActivity: PropTypes.func,
+  history: PropTypes.object
```

Comment:
you will have some problems here with changing to shape. So, generate reusable prop for route things. You can start with adding only a `push`, as at this moment you are using only the push. 
I will not give any more hints ;) handle it 💪